src/math/utils.rs

```diff
@@ -44,16 +44,22 @@ mod test {
     quickcheck! {
         fn resize_bounds_correctly_width(old_w: u32, new_w: u32) -> bool {
             if old_w == 0 || new_w == 0 { return true; }
+            if new_w as u64 * 400u64 >= old_w as u64 * u64::from(u32::MAX) { return true; }
+
```

Comment:
Did you mean to include this?

Reply:
Yes, that's the intent 13053fa5576d9fa1a6aa4ff1d1a032d298615dd5. Should I put the commit description as a comment for explanation of this part?